app/policies/application_authentication_policy.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationAuthenticationPolicy < DefaultPolicy
+  def create?
```

Comment:
Is there anyway to consolidate the contents of these 3 files into one via a mixin?

Just asking since the `create?`, and aliases methods are all duplicated.

Reply:
Oh I like that idea a lot. Implemented!